.env.example

```diff
@@ -0,0 +1,7 @@
+# Stripe keys
```

Comment:
@yuki-stripe I realized I forgot to include the `.env.example` file in the repo, might have forgotten this in the saving card repo too...